src/engraving/libmscore/measure.cpp

```diff
@@ -4057,11 +4061,13 @@ float Measure::durationStretch(Fraction curTicks, const Fraction minTicks) const
     // Logarithmic spacing (MS 3.6)
     //qreal str = 1.0 + 0.721 * slope * log(qreal(curTicks.ticks()) / qreal(minTicks.ticks()));
     // Quadratic spacing
-    qreal str = 1 - slope + slope * sqrt(qreal(curTicks.ticks()) / qreal(minTicks.ticks()));
+    // qreal str = 1 - slope + slope * sqrt(ratio);
+    // Custom spacing
+    double str = pow(slope, log2(ratio));
 
-    if (minTicks > longNoteThreshold) {
+    if (minSysTicks > longNoteThreshold) {
         // Avoids long notes being too narrow in the absense of shorter notes.
-        str = str * (1 - 0.5 + 0.5 * sqrt(qreal(minTicks.ticks()) / qreal(longNoteThreshold.ticks())));
+        str = str * (1 - 0.6 + 0.6 * sqrt(minSysTicks / longNoteThreshold));
```

Comment:
a few magic numbers in here, it might help to make named constants to help people who want to work on this code in the future! Or at least a comment explaining what these `0.6` values are.